site/en/tutorials/text/nmt_with_attention.ipynb

```diff
@@ -395,7 +395,7 @@
         "id": "rgCLkfv5uO3d"
       },
       "source": [
-        "### Create a tf.data dataset"
+        "### Use tf.data to create batches and shuffle the dataset"
```

Comment:
Unfortunately, the page nav isn't very wide, as you can see here: https://www.tensorflow.org/tutorials/text/nmt_with_attention
Open to changes that make this clear, but would like to avoid spilling over to a second line. Can always add further explanation to the text that follows.

Reply:
@lamberta I thought it's okay because it's already used in this page with the same page navigation: https://www.tensorflow.org/tutorials/generative/cvae#use_tfdata_to_create_batches_and_shuffle_the_dataset

Reply:
I'd rather include that page in this fix :)